ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1855,6 +1855,35 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4-kubevirt-hypershift-mce
+- as: aws-ipi-ovn-hypershift-mce-power-guest-critical-f7
+  cron: 17 20 7,14,23,30 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@ibmcloud-ipi and @critical and @hypershift-hosted'
+      MCE_VERSION: "2.4"
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;~NonPreRelease&;~HyperShiftMGMT&;~MicroShiftOnly&;~NonHyperShiftHOST&;~Serial&;~Disruptive&
+      TEST_IMPORTANCE: Critical
+      ZONES_COUNT: "3"
```

Comment:
`ZONES_COUNT` [defaults](https://github.com/openshift/release/pull/47527/files#diff-a526c7451dbe39493615b0d743ecf2d3d5bfa5c19105cf71effbfe5fb4077871R18) to 3, so not needed here.